collections/ansible_collections/osac/service/plugins/filter/find_template_roles.py

```
@@ -174,7 +176,7 @@ def validate_default(cls, value: Any) -> ProtobufAnyValue | None:
                 return ProtobufAnyValue(type=TypeMapping[type(value)], value=value)
             except KeyError as err:
                 raise ValueError(
-                    f"Default values must be scalar type, not {err}")
+                    f"Unsupported default value type: {err}")
```

Ruff (0.15.21)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)